### DISCUSSION TITLE
Location problem with an Android app

### DISCUSSION THREAD

**Discussion by veneilija** [MustardChef/WSABuilds#293] [Q&A]
First off all, thank you so much for this great WSA solution. 
I'm using WSA because I want to use a certain Android navigation app in a PC. I have a GPS device that connects to USB port. Some other Windows nav apps recognize the GPS and it works OK. From the Device Manager I can see that for this purpose Windows emulates USB port to COM port and those apps see it as COM.
However, the Android app that I'd like to use can't find the GPS. It only takes the default fixed location from Windows. I've tried to switch Windows location settings back and forth but with no success. If I prevent the location from Windows and click location on in the desired app it just keeps trying to get GPS but nothing happens.
Any ideas?

**Comment by MustardChef** [comment]
I would suggest installing the official version of WSA from the Microsoft Store and then reporting this issue to Microsoft via the Feedback Hub or via the WSA GitHub Repo.

**Reply by veneilija** [reply]
Unfortunately the official version is not available in Europe. I also found information from Microsoft support site telling that USB ports are not supported in the WSA. 
Now I'm purchasing a Galaxy S9 Ultra tablet with 14,6" display.

**Reply by veneilija** [reply]
I'm just wondering how WSA recognizes USB mouse and keyboard if USB ports are not supported as Microsoft says? How could I tell WSA that my USB GPS is actually a mouse?
